configuration/structures.go

```diff
@@ -92,15 +102,36 @@ func (cpf *chatPresenceFilter) WithUserIDs(user_ids []string, inclusive bool) *c
 
 // Bot represents basic bot agent information
 type Bot struct {
-	ID                   string         `json:"id"`
+	ID                   string        `json:"id"`
+	Name                 string        `json:"name,omitempty"`
+	AvatarPath           string        `json:"avatar_path,omitempty"`
```

Comment:
```suggestion
	AvatarPath           string        `json:"avatar,omitempty"`
```
One more thing, `avatar_path` json field is in v3.2 and v3.3, but it has been renamed to just `avatar` in newer versions. Please fix all the occurences in the SDK